### DISCUSSION TITLE
Adding prophage gene prediction and region extraction

### DISCUSSION THREAD

**Discussion by camilogarciabotero** [oschwengers/bakta#113] [Ideas]
Hi @oschwengers 

The improvements you have made to Bakta are awesome. Thank you.

Lately, I have been working on phages and while doing some research I've found that the [tools](https://github.com/voorloopnul/awesome-phages#gene-prediction) for phage annotation are poorly automated and require extensive manual tweaking. So I wondered if there is any way to integrate phage protein specialized annotation here and something like allowing extraction of a plausible region annotated?

Thanks,
Camilo.

**Comment by oschwengers** [comment]
Hi @camilogarciabotero , thanks for pointing out.
I'm keen to improve the annotation of all phage-related genes. However, as you already pointed out, there are many different tools/databases out there. To keep track, I've just updated this long-existing issue: https://github.com/oschwengers/bakta/issues/43

I'll let you know as soon as there's some update on this.

**Reply by camilogarciabotero** [reply]
Thank you @oschwengers It'll be awesome to see what comes next to Bakta!
